File: qlty-cli/tests/lang/php/basic.in/fields/class_declaration.php
<?php
  class Foo {
    public $bar = "";
    public $baz = "";

    public function __construct() {
      $this->bar = "";
      $this->baz = "";
    }
  }

  function doSomething() {
    $foo = new Foo();
    $foo->bar = "Hello";
    $foo->baz = "World";
    return ($foo->bar . $foo->baz);
  }
?>

File: qlty-cli/tests/lang/swift/basic.in/fields/ClassDeclaration.swift
// Class declaration with fields

class Foo {
    var bar: String
    var baz: String
    
    // Constructor
    init() {
        self.bar = ""
        self.baz = ""
    }
}

class ClassDeclaration {
    static func doSomething() -> String {
        let foo = Foo()
        foo.bar = "Hello"
        foo.baz = "World"
        return foo.bar + foo.baz
    }
    
    static func main() {
        print(doSomething())
    }
}